### ISSUE TITLE
causes fatal error when used with multisite and visiting the network admin page

### ISSUE THREAD

**Issue by anointed** [kaltura/all-in-one-video-pack.wordpress#2]
Setup:
WordPress 3.5alpha in multisite setup using folders

In the admin, there is a link to go to the network admin:
wp-admin/network/

This is where you manage all the sites in your network.

Problem:
If any site on the network has the all-in-one-video plugin activated, then when trying to visit the network admin page, you will get a blank page with a 500 error:

```
 mod_fcgid: stderr: PHP Fatal error:  Call to undefined function wp_get_current_user() in /home/mysite/public_html/wp-includes/capabilities.php on line 1261, referer: http://mysite.com/wp-admin/plugins.php?activate=true&plugin_status=all&paged=1&s=
```

I traced this error back to the  kaltura_comment_form() function where it says:

```
$user = wp_get_current_user();
```

Evidently this is not ok, and I guess it is due to this:
http://codex.wordpress.org/Function_Reference/wp_get_current_user

> Use the init or any subsequent action to call this function. Calling it outside of an action can lead to troubles. See #14024 for details. 

http://core.trac.wordpress.org/ticket/14024

Bottom line: You can't use the WordPress plugin on the new 3.5 WordPress without fatal errors in the network admin and possibly other areas.
